### PR TITLE
feature/frontend-history

### PR DESCRIPTION
**Pull request by paga16-hash**
change Manager -> Repository in controllers 
add API in log service to retrieve environment data
refactor consumer for log service similar to alarm
add frontend route, view and components to display history 
add delete button on notification 
add KafkaManager to monitoring service

### CODE REVIEW

File: frontend/src/utils/RequestHelper.ts [hunk 1]
```diff
@@ -12,12 +12,32 @@ type Headers = {
...
+  process.env.NODE_ENV == 'develop' ? '4001' : process.env.MONITORING_PORT || '4001'
+export const alarmHost: string =
+  process.env.NODE_ENV == 'develop' ? 'localhost' : process.env.ALARM_HOST || 'localhost'
+export const alarmPort: string = process.env.NODE_ENV == 'develop' ? '4002' : process.env.ALARM_PORT || '4002'
+
+export const logHost: string =
+  process.env.NODE_ENV == 'develop' ? 'localhost' : process.env.LOG_HOST || 'localhost'
+export const logPort: string = process.env.NODE_ENV == 'develop' ? '4003' : process.env.LOG_PORT || '4003'
+* */
+
```
Comment (Mala1180) [inline frontend/src/utils/RequestHelper.ts:32]: typo?

File: monitoring/src/utils/KafkaManager.ts
Comment (Mala1180) [inline monitoring/src/utils/KafkaManager.ts:1]: We should put the `KafkaManager` in commons..

**Review by Mala1180 — APPROVED** [review]
Good job!

File: monitoring/src/utils/KafkaManager.ts
Comment (paga16-hash) [reply monitoring/src/utils/KafkaManager.ts:1]: that's true

**Pull request merged**